CMakeLists.txt

```diff
@@ -239,6 +241,14 @@ else()
   find_package( hipblas-common REQUIRED CONFIG PATHS ${HIP_DIR} ${ROCM_PATH} /opt/rocm)
 endif()
 
+include(cmake/findRoctracer.cmake)
```

Comment:
Could you refer to https://github.com/ROCm/MIOpen/blob/61d9825a5eaec632a727e30d4ea881f28c38626f/CMakeLists.txt#L529-L530 and https://github.com/ROCm/MIOpen/blob/61d9825a5eaec632a727e30d4ea881f28c38626f/CMakeLists.txt#L533

Reply:
Fixed